{
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1",
    "cveMetadata": {
        "cveId": "CVE-2022-43902",
        "assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
        "state": "PUBLISHED",
        "assignerShortName": "ibm",
        "dateReserved": "2022-10-26T15:46:22.841Z",
        "datePublished": "2023-03-01T20:44:15.672Z",
        "dateUpdated": "2024-07-04T11:59:33.798Z"
    },
    "containers": {
        "cna": {
            "affected": [
                {
                    "defaultStatus": "unaffected",
                    "product": "MQ",
                    "vendor": "IBM",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "9.2 CD, 9.2 LTS, 9.3 CD, 9.3 LTS"
                        }
                    ]
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "supportingMedia": [
                        {
                            "base64": false,
                            "type": "text/html",
                            "value": "IBM MQ 9.2 CD, 9.2 LTS, 9.3 CD, and 9.3 LTS is vulnerable to a denial of service attack caused by specially crafted PCF or MQSC messages.  IBM X-Force ID:  240832."
                        }
                    ],
                    "value": "IBM MQ 9.2 CD, 9.2 LTS, 9.3 CD, and 9.3 LTS is vulnerable to a denial of service attack caused by specially crafted PCF or MQSC messages.  IBM X-Force ID:  240832."
                }
            ],
            "metrics": [
                {
                    "cvssV3_1": {
                        "attackComplexity": "LOW",
                        "attackVector": "NETWORK",
                        "availabilityImpact": "HIGH",
                        "baseScore": 6.5,
                        "baseSeverity": "MEDIUM",
                        "confidentialityImpact": "NONE",
                        "integrityImpact": "NONE",
                        "privilegesRequired": "LOW",
                        "scope": "UNCHANGED",
                        "userInteraction": "NONE",
                        "vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
                        "version": "3.1"
                    },
                    "format": "CVSS",
                    "scenarios": [
                        {
                            "lang": "en",
                            "value": "GENERAL"
                        }
                    ]
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "703 Improper Check or Handling of Exceptional Conditions",
                            "lang": "en"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "orgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
                "shortName": "ibm",
                "dateUpdated": "2023-03-10T20:04:40.537099Z"
            },
            "references": [
                {
                    "tags": [
                        "vendor-advisory"
                    ],
                    "url": "https://www.ibm.com/support/pages/node/6890643"
                },
                {
                    "tags": [
                        "vdb-entry"
                    ],
                    "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/240832"
                }
            ],
            "source": {
                "discovery": "UNKNOWN"
            },
            "title": "IBM MQ denial of service",
            "x_generator": {
                "engine": "Vulnogram 0.1.0-dev"
            }
        },
        "adp": [
            {
                "providerMetadata": {
                    "orgId": "96f1cf3c-f80a-4e0f-a063-99a91e3f1325",
                    "shortName": "secretariat-reference",
                    "dateUpdated": "2024-07-04T11:59:33.798Z"
                },
                "references": [
                    {
                        "tags": [
                            "vendor-advisory",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://www.ibm.com/support/pages/node/6890643/DEMO_2024_06_19"
                    },
                    {
                        "tags": [
                            "vdb-entry",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/240832/DEMO_2024_06_19"
                    },
                    {
                        "tags": [
                            "vendor-advisory",
                            "x_snapshot"
                        ],
                        "url": "https://www.ibm.com/support/pages/node/6890643/adp-test-6-28"
                    },
                    {
                        "tags": [
                            "vdb-entry",
                            "x_snapshot"
                        ],
                        "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/240832/adp-test-6-28"
                    }
                ]
            }
        ]
    }
}